geode-protobuf/src/main/java/org/apache/geode/internal/protocol/protobuf/ProtobufStreamProcessor.java

```diff
@@ -74,6 +71,10 @@ private void processOneMessage(InputStream inputStream, OutputStream outputStrea
     statistics.messageReceived(message.getSerializedSize());
 
     ClientProtocol.Request request = message.getRequest();
+    if (message.getMessageTypeCase()
+        .getNumber() == ClientProtocol.Request.SIMPLEAUTHENTICATIONREQUEST_FIELD_NUMBER) {
+      // auith
```

Comment:
`auth` maybe? Also, are we missing some implementation? Or is this just a legacy check that used to have logic attached to it?

Reply:
Hmm, I think this was part of an aborted effort to move the authentication call into the stream processor instead of the pipeline.  I'll double check with Galen to make sure this should go away.